src/app/admin/deleteUser/DeleteUserDialog.tsx

```diff
@@ -48,17 +53,8 @@ const DeleteUserDialog: React.FC<Props> = (props) => {
                 ),
             });
             void logInfoReturnLogId(`${props.userToDelete?.email} deleted successfully.`);
-        } else {
-            const logId = await logErrorReturnLogId(
-                `Error with delete: User ${props.userToDelete.email}`,
-                { error }
-            );
```

Comment:
We should still log an error

Reply:
I've sent a log within adminDeleteUser and returned the logID to be displayed